components/locale_canonicalizer/src/provider.rs

```diff
@@ -25,13 +25,13 @@ type UnvalidatedSubdivision = TinyAsciiStr<7>;
 // LanguageIdentifier doesn't have an AsULE implementation, so we have
 // to store strs and parse when needed.
 type UnvalidatedLanguageIdentifier = str;
-type UnvalidatedLanguageIdentifierPair<'data> = StrStrPair<'data>;
+type UnvalidatedLanguageIdentifierPair<'data> = StrStrPairVarULE;
```

Comment:
no lifetime

Reply:
the fact that this even compiles is one of rust's weird quirks around type aliases but pretend that it doesn't.